packages/broker/test/integration/config.test.ts

```diff
@@ -1,13 +1,22 @@
 import fs from 'fs'
 import path from 'path'
 import { createBroker } from '../../src/broker'
+import { DEFAULT_ENTRYPOINTS } from '../utils'
 
 const PATH = './configs'
 
 describe('Config', () => {
 
     it('start with minimal config', async () => {
-        const broker = await createBroker({})
+        const broker = await createBroker({
```

Comment:
Testing client?

Reply:
Not really, as Broker also has other configuration options, which are not related to the client: https://github.com/streamr-dev/network/blob/bd1b7ef5f0beca27b11918fccdf7f648a3bfaca1/packages/broker/src/config/config.ts#L8